hack/version2.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+declare -r UNKNOWN_BRANCH_PREFIX="0.0"
+declare -r MASTER_BRANCH_PREFIX="1.2"
+
+function prefix_from_branch() {
+	declare -r branch_name="$1"
+
+	if [[ "$branch_name" =~ ^([^/]+/)?mce-([[:digit:]]+.[[:digit:]]+) ]]; then
+		echo "${BASH_REMATCH[2]}"
+	elif [[ "$branch_name" =~ ^([^/]+/)?master$ ]]; then
+		echo "$MASTER_BRANCH_PREFIX"
+	else
+		echo "$UNKNOWN_BRANCH_PREFIX"
+	fi
+}
+
+function commit_count() {
+	git rev-list --count "$(git rev-list --max-parents=0 HEAD)..HEAD"
+}
+
+function short_sha() {
+	git rev-parse --short HEAD
```

Comment:
To be compatible with the existing functionality, this should be 
```suggestion
	git rev-parse --short=7 HEAD
```